packages/vm/src/context.rs

```diff
@@ -19,14 +19,23 @@ pub use iter_support::{
 use crate::errors::Error;
 use crate::memory::{read_region, write_region};
 
+static MAX_LENGTH_DB_KEY: usize = 100_000;
```

Comment:
I would limit DB keys to 1000 max. That already starts to become a problem for some stores, and no need to have those big, no matter how many objects you shove into an array